core/src/test/java/com/scalar/db/storage/cosmos/CosmosOperationCheckerTest.java

```diff
@@ -397,6 +418,284 @@ public void check_ForMutationsWithDeleteWithCondition_ShouldBehaveProperly() {
         .isInstanceOf(IllegalArgumentException.class);
   }
 
+  @Test
+  public void
+      check_GetGiven_WhenIllegalCharacterInPrimaryKeyColumn_ShouldThrowIllegalArgumentException()
+          throws ExecutionException {
+    // Arrange
+    when(metadataManager.getTableMetadata(any())).thenReturn(TABLE_METADATA2);
+
+    Get get1 =
+        Get.newBuilder()
+            .namespace(NAMESPACE_NAME)
+            .table(TABLE_NAME)
+            .partitionKey(Key.ofText(PKEY1, "ab"))
+            .clusteringKey(Key.ofText(CKEY1, "ab"))
+            .build();
+    Get get2 =
+        Get.newBuilder()
+            .namespace(NAMESPACE_NAME)
+            .table(TABLE_NAME)
+            .partitionKey(Key.ofText(PKEY1, "a:b"))
+            .clusteringKey(Key.ofText(CKEY1, "ab"))
+            .build();
+    Get get3 =
+        Get.newBuilder()
+            .namespace(NAMESPACE_NAME)
+            .table(TABLE_NAME)
+            .partitionKey(Key.ofText(PKEY1, "ab"))
+            .clusteringKey(Key.ofText(CKEY1, "a:b"))
+            .build();
+    Get get4 =
+        Get.newBuilder()
+            .namespace(NAMESPACE_NAME)
+            .table(TABLE_NAME)
+            .partitionKey(Key.ofText(PKEY1, "a/b"))
+            .clusteringKey(Key.ofText(CKEY1, "ab"))
+            .build();
+    Get get5 =
+        Get.newBuilder()
+            .namespace(NAMESPACE_NAME)
+            .table(TABLE_NAME)
+            .partitionKey(Key.ofText(PKEY1, "ab"))
+            .clusteringKey(Key.ofText(CKEY1, "a/b"))
+            .build();
+    Get get6 =
+        Get.newBuilder()
+            .namespace(NAMESPACE_NAME)
+            .table(TABLE_NAME)
+            .partitionKey(Key.ofText(PKEY1, "a\\b"))
```

Comment:
```suggestion
            .partitionKey(Key.ofText(PKEY1, "a\b"))
```
Ditto. I think we should validate `\` (single backslash) instead of `\\` (double backslashes).